### ISSUE TITLE
KeepAlive HTTPS Connections

### ISSUE THREAD

**Issue by KraigM** [aws/aws-sdk-net#255]
From what I can tell at least, you don't seem to be using HTTPS persistent connections.  This seems problematic especially if you are making several calls to services.  I know PCL does not support it but the Core is platform specific and each platform supports it.  From what I understand, it is as simple as setting KeepAlive to true and .net takes care of the rest (https://msdn.microsoft.com/en-us/library/system.net.httpwebrequest.keepalive.aspx).  Is there a reason you are not leveraging this?  Is there some way to at least enable it upon request?

**Comment by normj** [comment]
KeepAlive is on by default with HttpWebRequest. The documentation you linked to points that out in the Property Value section. Are you seeing some behavior that indicates that persistent connections are not being used?

**Comment by KraigM** [comment]
Yeah I did several checks to prove this, but apparently it was Charles Proxy who was causing the SSL handshake to happen every time (despite their claims that persistent connections are supported).  I did notice that the iOS app will reestablish the connection after one or two reuses, but that might be something with Xamarin. It continually reuses the connection on mono (dropping from 1s end-to-end to .2s end-to-end per request).  Thanks for your help anyway!
